ee/src/main/java/org/jboss/as/ee/component/deployers/InterceptorAnnotationProcessor.java

```diff
@@ -82,7 +82,7 @@ private void processComponentConfig(final EEApplicationClasses applicationClasse
         final Class<?> componentClass;
         try {
             componentClass = ClassLoadingUtils.loadClass(description.getComponentClassName(), deploymentUnit);
-        } catch (ClassNotFoundException e) {
+        } catch (Exception e) {
```

Comment:
Should this not be `Throwable` rather than `Exception`? The two failures reported in WFLY-8515 were `LinkageError` and `NoClassDefFoundError`, neither of which would be caught as things stand